services/payments/src/simcore_service_payments/api/rpc/_payments_methods.py

```diff
@@ -127,6 +128,11 @@ async def init_payment_with_payment_method(  # noqa: PLR0913 # pylint: disable=t
     user_email: EmailStr,
     comment: str | None = None,
 ) -> WalletPaymentInitiated:
+    warnings.warn(
+        f"{__name__}.init_payment_with_payment_method is deprecated. Use instead pay_with_payment_method",
```

Comment:
is this method still needed? if not I would prefer to see it disappear. if it will disappear later, then maybe add in the warning when it is safe to do so